.github/workflows/scripts/go-coverage.sh

```diff
@@ -10,7 +10,7 @@ tests
 test
 EndOfMessage
 
-PASSPERCENT=40
+PASSPERCENT=30
```

Comment:
I think it should be down to 20 because we are still developing GMC, it may exhaust us many efforts for focusing on unit test, does it make sense?

Reply:
check